controllers/suite_test.go

```diff
@@ -133,6 +142,21 @@ var _ = BeforeSuite(func(done Done) {
 	client = mgr.GetClient()
 	Expect(client).ToNot(BeNil())
 
+	// used in schema-replication-controller test
+	secret := corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "endpoints-secret",
+			Namespace: "default",
+		},
+		Type: corev1.SecretTypeOpaque,
+		Data: map[string][]byte{
+			"username":  []byte("a-random-user"),
+			"password":  []byte("a-random-password"),
+			"endpoints": []byte("a.endpoints.local:5672,b.endpoints.local:5672,c.endpoints.local:5672"),
```

Comment:
Do we want to leave this as a single comma-separated text field, or would it make more sense for it to be an array?

Reply:
It's a secret object. I don't think you can have anything other than string as the value.